geode-junit/src/main/java/org/apache/geode/codeAnalysis/AnalyzeDataSerializablesTestBase.java

```diff
@@ -47,18 +46,17 @@
 import org.apache.geode.internal.serialization.BufferDataOutputStream;
 import org.apache.geode.internal.serialization.KnownVersion;
 import org.apache.geode.test.junit.categories.SerializationTest;
-import org.apache.geode.test.junit.rules.ClassAnalysisRule;
 
 /**
  * This abstract test class is the basis for all of our AnalyzeModuleNameSerializables tests.
  * Subclasses must provide serialization/deserialization methods.
  *
  * <p>
- * Most tests should subclass {@link AnalyzeSerializablesJUnitTestBase} instead of this
+ * Most tests should subclass {@link AnalyzeSerializablesTestBase} instead of this
```

Comment:
This javadoc should now reference the new `AnalyzeSerializablesWithClassAnalysisRuleTestBase`.

Reply:
Done